tests/test_inputmedia.py

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env python
+#
+# A library that provides a Python interface to the Telegram Bot API
+# Copyright (C) 2015-2017
+# Leandro Toledo de Souza <devs@python-telegram-bot.org>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see [http://www.gnu.org/licenses/].
+import pytest
+from flaky import flaky
+
+from telegram import InputMediaVideo, InputMediaPhoto, Message
+from .test_video import video, video_file
+from .test_photo import _photo, photo_file, photo, thumb
+
+
+@pytest.fixture(scope='class')
+def input_media_video():
+    return InputMediaVideo(media=TestInputMediaVideo.media,
+                           caption=TestInputMediaVideo.caption,
+                           width=TestInputMediaVideo.width,
+                           height=TestInputMediaVideo.height,
+                           duration=TestInputMediaVideo.duration)
+
+
+@pytest.fixture(scope='class')
+def input_media_photo():
+    return InputMediaPhoto(media=TestInputMediaPhoto.media,
+                           caption=TestInputMediaPhoto.caption)
+
+
+class TestInputMediaVideo(object):
+    type = "video"
+    media = "NOTAREALFILEID"
+    caption = "My Caption"
+    width = 3
+    height = 4
+    duration = 5
+
+    def test_expected_values(self, input_media_video):
+        assert input_media_video.type == self.type
+        assert input_media_video.media == self.media
+        assert input_media_video.caption == self.caption
+        assert input_media_video.width == self.width
+        assert input_media_video.height == self.height
+        assert input_media_video.duration == self.duration
+
+    def test_to_dict(self, input_media_video):
+        input_media_video_dict = input_media_video.to_dict()
+        assert input_media_video_dict['type'] == input_media_video.type
+        assert input_media_video_dict['media'] == input_media_video.media
+        assert input_media_video_dict['caption'] == input_media_video.caption
+        assert input_media_video_dict['width'] == input_media_video.width
+        assert input_media_video_dict['height'] == input_media_video.height
+        assert input_media_video_dict['duration'] == input_media_video.duration
+
+    def test_with_video(self, video):
+        # fixture found in test_video
+        imv = InputMediaVideo(video, caption="test 3")
```

Comment:
I don't think we use abbreviations in the other tests

Reply:
We don;t and I'm lazy, will fix